doc/introduction.rst

```diff
@@ -17,54 +17,114 @@
+- Clip resizing, scaling, trimming, snapping, rotation, and cutting
+- Compositing, image overlays, watermarks
+- Cross-platform (Supports Linux, macOS, and Windows)
+- Desktop integration (Drag & Drop support)
+- Digital video effects (including brightness, gamma, hue, and gray-scale)
+- Frame accuracy (step through each frame of video)
+- Powerful curve-based Key frame animations
+- Scrolling motion picture credits
+- Support for many `video, audio, and image formats 
+  <https://en.wikipedia.org/wiki/FFmpeg#Supported_codecs_and_formats>`_
+- SVG friendly, to create and include vector titles and credits
+- Time-mapping and speed changes on clips (slow/fast, forward/backward, etc.)
+- Title templates, title creation, sub-titles
+- Unlimited tracks/layers for video editing
+- Video transitions with real-time previews
+- and many more!
 
 Screenshot
 ----------
-.. image:: images/ui-example.jpg
+
+.. image:: images/ui-example.png
 
 System Requirements
 -------------------
-Video editing benefits from large amounts of memory, modern CPUs, and fast disks. Basically,
-you want the best computer you can afford when editing video. Here are the minimum system
-requirements:
 
-* 64-bit Operating System (Linux, OS X, Windows 7/8/10)
-* Multi-core processor with 64-bit support
-* 4GB of RAM (16GB recommended)
-* 500 MB of hard-disk space for installation
-* Optional: Solid-state drive (SSD), if utilizing disk-caching (and an additional 10GB of
-  hard-disk space)
+Video editing uses sizable amounts of memory, modern multi-core CPUs, and 
+fast internal storage.  It is always best to have as much memory as possible.  
+You will also want a powerful processor.  You can manage using any processor 
+with a speed of 2.4 gigahertz (GHz) or higher.  Finally, you will want to make 
+sure you have plenty of storage available to hold the software and video 
+with which you will be working.  A Solid State Device (SSD) provides faster 
+disk access which helps when finalizing your project.  Most modern computers 
+should work well with OpenShot™.
+
+When editing video with OpenShot™, you want a computer system that meets these 
+**minimum system requirements**:
+
+- 64-bit Multi-core processor (2.4+ GHz)
+- 4 GB of Random-Access Memory (RAM)
+- 500 MB of free storage space for installation
+- 500 GB of internal storage space for processing
+- 64-bit Operating System (Linux, macOS, Windows 64-bit)
+- Monitor or laptop display capable of 1280x720 (720P) resolution
+
+A more in depth version of the System Requirements is available in the 
+`User Manual <manual.html#system-requirements>`_.
 
 License
 -------
-OpenShot Video Editor is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 3 of the License, or
-(at your option) any later version.
+
+OpenShot™ Video Editor is Free and Open Source Software (FOSS).  It is fully
```

Comment:
We should be careful about changing the wording of our license from GNU. I would stick with there official wording, and not freestyle with our legal sentences. :wink